EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs

```diff
@@ -74,35 +74,60 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                     new(OpCodes.Brfalse, returnLabel),
                 });
 
-            // This is to find the location of RpcMakeSound to remove the original code and add a new sever logic structure (Start point)
-            int addShouldSeverOffset = -1;
-            int addShouldSeverIndex = newInstructions.FindLastIndex(
-                instruction => instruction.Calls(Method(typeof(Scp330Interobject), nameof(Scp330Interobject.RpcMakeSound)))) + addShouldSeverOffset;
+            /* next code will used to override sound rpc check by EXILED
+             * old:
+             *   if (args.PlaySound)
+             * new:
+             *   if (args.PlaySound || ev.PlaySound)
```

Comment:
`if (args.PlaySound || ev.PlaySound)`

So if you reject it in Exiled, this check is still going to allow play sound?

Reply:
Oh, yes, i was trying to make that more nwapi supportive, but ill change that to `&&`